src/content/docs/alerts-applied-intelligence/transition-guide.mdx

```diff
@@ -1,15 +1,79 @@
+  >
+
+  ## Analyze [#analyze]
+
+The [overview page](/docs/alerts-applied-intelligence/new-relic-alerts/get-started/alerts-ai-overview-page) shows your most recent open, closed, and muted violations.
+
+The **Issues & activity** page shows your most recent [issues](/docs/alerts-applied-intelligence/new-relic-alerts/get-started/alerts-ai-overview-page/#issues), [incidents](/docs/alerts-applied-intelligence/new-relic-alerts/get-started/alerts-ai-overview-page/#incidents), and [anomalies](/docs/alerts-applied-intelligence/new-relic-alerts/get-started/alerts-ai-overview-page/#anomalies).
+
+## Detect [#detect]
+
+The [alert conditions (policies) page](/docs/alerts-applied-intelligence/new-relic-alerts/alert-policies/create-edit-or-find-alert-policy/) lists the policies where your conditions are created and organized. Create new policies here.
+
+The [anomaly detection page](/docs/alerts-applied-intelligence/applied-intelligence/proactive-detection/expanded-anomaly-detection/) lists your expanded anomaly detection configurations. You can add or edit the configurations here.
+
+## Correlate [#correlate]
+
+The [sources page](/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/get-started-incident-intelligence/#1-configure-sources) shows your configured incident intelligence sources. You can add new sources or configure existing ones.
+
+The [decisions page](/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions/) shows your correlated incidents. You can review, edit, and add decisions.
+
+## Enrich & notify [#enrich]
+
+Use the [muting rules page](/docs/alerts-applied-intelligence/new-relic-alerts/alert-notifications/muting-rules-suppress-notifications/) to temporarily stop your alerts violation notifications.
+
+Use [workflows](/docs/alerts-applied-intelligence/applied-intelligence/incident-workflows/incident-workflows/) to add more data to your alerts notifications.
+
+Use [destinations](/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/get-started-incident-intelligence/#2-configure-destinations) to configure where your applied intelligence notifications get sent.
+
+Use [pathways](/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/get-started-incident-intelligence/#3-configure-pathways) to configure where your correlated issues get sent.
+
+Use [incident workflows](/docs/alerts-applied-intelligence/applied-intelligence/incident-workflows/incident-workflows/) to automate common problems and actions related to your alerts policies.
+
+Use [incident destinations](/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/incident-intelligence-destination-examples/) to add and configure third-party services for your incidents.
+
+## Settings [#settings]
+
+Use the settings page to review and manage your current incident intelligence usage.
+
+## Alerts (classic) [#alerts]
+
```

Comment:
I would like the intro here to say basically what you told me when we were chatting, that this is the existing UI and there's nothing wrong with that, it's just a different view while we transition the entire UI. Or something along those lines.